src/cdk/overlay/position/flexible-connected-position-strategy.ts

```diff
@@ -38,6 +38,9 @@ export type FlexibleConnectedPositionStrategyOrigin = ElementRef | Element | Poi
   height?: number;
 };
 
+/** Equivalent of `ClientRect` without some of the properties we don't care about. */
+type Dimensions = Omit<ClientRect, 'x' | 'y' | 'toJSON'>;
```

Comment:
I wonder if it would make sense to have an explicit type (with properties defined) in `overlay/position` instead of doing it that way. Not feeling strongly, just an idea.

Reply:
I went with this approach since it's still less code than listing out the properties individually on an interface.